### PR TITLE
Updates to cards in Resources section

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Fixes https://github.com/red-hat-data-services/odh-dashboard/issues/23

image

### CODE REVIEW

**Comment by kywalker-rh** [comment]
The icon for the In Progress label should be updated to the small `fa-sync-alt` (SyncAltIcon) icon. Otherwise everything looks great!

**Comment by jeff-phillips-18** [comment]
@kywalker-rh Updated:

image

**Review by kywalker-rh — APPROVED** [review]

File: frontend/src/components/OdhDocCard.tsx [hunk 1]
```diff
@@ -34,46 +38,67 @@ const OdhDocCard: React.FC<OdhDocCardProps> = ({ odhDoc }) => {
...
+            </Tooltip>
+            <div className="odh-card__partner-badge odh-m-doc odh-m-duration">
+              {odhDoc.spec.durationMinutes} minutes
+            </div>
           </div>
+          {isQuickStartInProgress(quickStart.metadata.name, qsContext) ? (
+            <div className="odh-card__doc-badges .m-progress">
+              <div className="odh-card__partner-badge odh-m-doc odh-m-in-progress">
+                <SyncAltIcon />
+                In Progress
```
Comment (jschuler) [inline frontend/src/components/OdhDocCard.tsx:85]: To align more with the default quick starts catalog library, could be sentence case
```suggestion
                In progress
```

Default quick starts catalog as of now also displays the labels in the following order:
1. (Optional) type/category - i.e. Quick start / Documentation
2. In progress / Complete
3. Duration

Comment (jeff-phillips-18) [reply frontend/src/components/OdhDocCard.tsx:85]: Are the horizontal? Could you attach a screen shot of a card with all three?

Comment (jschuler) [reply frontend/src/components/OdhDocCard.tsx:85]: At the moment it looks like this:
Screen Shot 2021-03-23 at 3 35 12 PM

But we can change it if it makes more sense in a different order.
cc @jgiardino

Comment (jschuler) [reply frontend/src/components/OdhDocCard.tsx:85]: I'll change the order to follow suit with your order:
1. Type
2. Duration
3. In progress / complete

Comment (jgiardino) [reply frontend/src/components/OdhDocCard.tsx:85]: 👍 I also like the order that RHODS currently uses.

**Comment by jeff-phillips-18** [comment]
Updated:
image

 @jschuler PTAL

**Review by jschuler — APPROVED** [review]

**Pull request merged**